scripts/jira/comment-with-review-app-url.js

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env node
+const axios = require('axios');
+const JIRA_API_VERSION = '2';
+const JIRA_API_URL = `https://1024pix.atlassian.net/rest/api/${JIRA_API_VERSION}`;
+
+async function main() {
+  // eslint-disable-next-line eqeqeq
+  const isReviewApp = process.env.REVIEW_APP == 'true';
```

Comment:
```suggestion
  const isReviewApp = process.env.REVIEW_APP === 'true';
```
Question: Y a t-il une raison particulière à ne pas utiliser l'égalité type-safe ?

Reply:
C'est un copier/coller des autres repo, j'ai pas la réponse désolé :/